README.md

```diff
@@ -16,28 +16,32 @@ This toolkit is built on top of Azure Batch but does not require any Azure Batch
 - Ability to run _spark submit_ directly from your local machine's CLI
 
 ## Setup
+
 1. Clone the repo
```

Comment:
Are we comfortable enough to just pip install? Technically it should be the same thing, right??

Reply:
technically, yes. But I elected not to make that change here until our release on pip is "official". I can do it now if you'd prefer.